helm-chart/sefaria-project/values.yaml

```diff
@@ -235,6 +235,34 @@ varnish:
     image: ibmcom/varnish-metrics-exporter
     tag: 0.32.0
 
+tasks:
+  enabled: false
+  resources:
+    requests:
+      memory: "100Mi"
```

Comment:
@BrendanGalloway these resources may be too low but we'll see once the pod comes up.

Reply:
ack, just put in random values for now